src/service/print/csv.js

```diff
@@ -380,10 +380,10 @@ async function renderMCQ(
     Subject: subject,
     QuestionSetName: examName,
     Questions: finalQuestion,
-    Option1: questionOptions.length !== 0 ? questionOptions[0][0] : "",
-    Option2: questionOptions.length !== 0 ? questionOptions[1][0] : "",
-    Option3: questionOptions.length !== 0 ? questionOptions[2][0] : "",
-    Option4: questionOptions.length !== 0 ? questionOptions[3][0] : "",
+    Option1: questionOptions.length > 0 ? questionOptions[0][0] : "",
```

Comment:
I would suggest to use for loop instead of manualy coding for each option

Reply:
Use for loop for number of options & append the key(option+i) to the "data" object.